packages/label-sync/src/labels.json

```diff
@@ -244,6 +244,36 @@
       "name": "stale: extraold",
       "description": "Pull request is critically old and needs prioritization.",
       "color": "801904"
+    },
+    {
```

Comment:
Do we want to have the labels sync'ed everywhere? This will sync on more repos than auto-label is installed/enabled on.

An alternative is to use `label-utils` (https://github.com/googleapis/repo-automation-bots/tree/main/packages/label-utils#usage-1) in a repository cron which would also allow conditionally adding the labels iff that feature is enabled.

Reply:
OK, makes sense

Reply:
The extra calls are negligible compared to the number of webhook events we handle. If we for sure want these on all our repos, then we can do it, but we sometimes do get complaints for being heavy-handed in repositories that haven't necessarily opted in.

Reply:
Makes complete sense - will add new code to `schedule.repository` event so it will be less `syncLabel` calls